### ISSUE TITLE
Xamarin quickstart "Performing Navigation" page results in inaccessible listview

### ISSUE THREAD

**Issue by zersiax** [MicrosoftDocs/xamarin-docs#3261]
If one were to follow the instructions in this article verbatim, the resulting UWP version of the app will have an accessibility issue.
Screenreaders will be unable to infer what note is selected in the collectionview. Instead, they will receive the following output:
Xamarin.Forms.Platform.UWP.ItemTemplateContext

I am not well-versed enough with either Xamarin, WPF or XAML to suggest what the best fix would be, but I'd say it's probably a good idea to give the right example to aspiring developers.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7dc72705-2668-156f-d214-1a35d5a2ea4d
* Version Independent ID: 993ba392-f860-1a29-7d62-0dc1efbf6f0e
* Content: [Perform navigation in a Xamarin.Forms application - Xamarin](https://docs.microsoft.com/en-us/xamarin/get-started/quickstarts/navigation?pivots=windows)
* Content Source: [docs/get-started/quickstarts/navigation.md](https://github.com/MicrosoftDocs/xamarin-docs/blob/live/docs/get-started/quickstarts/navigation.md)
* Product: **xamarin**
* Technology: **xamarin-forms**
* GitHub Login: @davidbritch
* Microsoft Alias: **dabritch**

**Comment by davidbritch** [comment]
Hi @zersiax 

On UWP, Shell and CollectionView are only partially implemented, hence your problem. This is why the quickstarts deliberately focus on building mobile apps using iOS and Android, and have no coverage of UWP.

Xamarin is currently a mobile platform, and the desktop support is sometimes lacking. That will change in MAUI when desktop support will become fully fledged.

I realise this isn't what you may want to hear, but I'd rather be honest about it.

**Comment by zersiax** [comment]
No problem at all :)  I was already attempting to manually override the automation properties without much luck, so hearing that it isn't my fault this isn't working actually is good to  hear :)
I haven't had a chance to test this on a mobile device yet, but will at some point soon. I'll report if I find gotchas on the supported platforms and keep an eye on MAUI development :)
